lib/enkf/config_keys.cpp

```diff
@@ -46,18 +46,6 @@ const char * config_keys_get_queue_option_key() {
   return QUEUE_OPTION_KEY;
 }
 
-const char * config_keys_get_lsf_resources_key() {
```

Comment:
should these names also be removed from config_keys.hpp?

Reply:
Yes! good catch :+1: